server/routes/create-an-activity/handlers/checkAnswers.ts

```diff
@@ -40,7 +63,104 @@ export default class CheckAnswersRoutes {
       })),
     } as ActivityCreateRequest
 
-    const response = await this.activitiesService.createActivity(activity, user)
+    const createActivityResponse = await this.activitiesService.createActivity(activity, user)
+
+    const slots = this.mapSlots(createJourney)
+
+    const activitySchedule = {
+      description: createJourney.name,
+      startDate: formatDate(plainToInstance(SimpleDate, createJourney.startDate).toRichDate(), 'yyyy-MM-dd'),
+      endDate: createJourney.endDate
+        ? formatDate(plainToInstance(SimpleDate, createJourney.endDate).toRichDate(), 'yyyy-MM-dd')
+        : undefined,
+      locationId: createJourney.location.id,
+      capacity: createJourney.capacity,
+      slots,
+      runsOnBankHoliday: createJourney.runsOnBankHoliday,
+    } as ActivityScheduleCreateRequest
+
+    const response = await this.activitiesService.createScheduleActivity(
+      createActivityResponse.id,
+      activitySchedule,
+      user,
+    )
```

Comment:
Just wanted to confirm this this is the initial two-call way, and that we might combine these at the API level, so we get one transaction which commits (or fails) for both Activity and Activity schedule? Appreciate you're working with what is available at present.